pages/linux/nautilus.md

```diff
@@ -0,0 +1,13 @@
+# nautilus
+
+> Default file explorer for Gnome desktop environment.
+> Also known as Gnome Files.
+> More information: <https://manned.org/nautilus>.
+
+- Launch Nautilus:
+
+`nautilus`
+
+- Launch Nautilus as root user:
+
+`sudo nautilus`
```

Comment:
I'm not sure about this example, because it seems that sudo launches a command as root.  @sbrl

Reply:
It shows, that Nautilus can be launched as root, which can be good to know. This isn't possible for Dolphin for example.

Reply:
👍🏻